metacontroller_pilot/metacontroller_pilot/metacontroller_sim.py

```diff
@@ -70,47 +70,42 @@ def main(args=None):
     print ("------------------------------")
     print ("Specify the option number:")
     print ("------------------------------")
+    print (" 0) Default")
     print (" 1) Battery low")
-    print (" 2) Network down")
-    print (" 3) Robot lost")
-    print (" 4) Obstructed")
+    print (" 2) Robot lost")
+    print (" 3) Obstructed")
     print ("------------------------------")
-    print (" 5) Charge completed")
-    print (" 6) Network reset")
-    print (" 7) Robot located")
-    print (" 8) Obstructed solved")
-    print (" 9) Laser Error")
```

Comment:
Hi @jginesclavero 

Why don't select directly the mode instead of the event?

Reply:
You are right, I fixed it in my last commit